mars/dataframe/groupby/aggregation.py

```diff
@@ -293,6 +298,102 @@ def __call__(self, groupby):
         else:
             return self._call_series(groupby, df)
 
+    @classmethod
+    def partition_merge_data(cls, op, partition_chunks, proxy_chunk, in_df):
```

Comment:
Could you add type hints for arguments? Only add hints for the new functions that you add. Due to historical reason,  some functions do not have type hints, but we hope that all the new functions should have.

Reply:
So do the other new functions.

Reply:
Fixed @qinxuye. Please confirm.